packages/social-previews/src/facebook-preview/link-preview.tsx

```diff
@@ -0,0 +1,44 @@
+import { compact } from 'lodash';
+import { firstValid, hardTruncation, shortEnough, stripHtmlTags } from '../helpers';
+import type { FacebookPreviewProps } from './types';
+
+const TITLE_LENGTH = 80;
+const DESCRIPTION_LENGTH = 200;
+
+const baseDomain = ( url: string ): string =>
+	url &&
+	url
+		.replace( /^[^/]+[/]*/, '' ) // strip leading protocol
+		.replace( /\/.*$/, '' ); // strip everything after the domain
+
+const facebookTitle = firstValid( shortEnough( TITLE_LENGTH ), hardTruncation( TITLE_LENGTH ) );
+
+const facebookDescription = firstValid(
+	shortEnough( DESCRIPTION_LENGTH ),
+	hardTruncation( DESCRIPTION_LENGTH )
+);
+
+const FacebookLinkPreview: React.FC< FacebookPreviewProps > = ( props ) => {
+	const { url, title, description, image, author } = props;
+
+	return (
+		<div className="facebook-preview">
+			<div className="facebook-preview__content">
+				<div className="facebook-preview__image">
+					{ image && <img alt="Facebook Preview Thumbnail" src={ image } /> }
+				</div>
+				<div className="facebook-preview__body">
+					<div className="facebook-preview__url">
+						{ compact( [ baseDomain( url ), author ] ).join( ' | ' ) }
```

Comment:
If I remember correctly, there have been discussions about removing/reducing the dependency on lodash. So, it may be better to use native functions:
```suggestion
						{ [ baseDomain( url ), author ].filter( Boolean ).join( ' | ' ) }
```

Reply:
I've just copied over the code from the current implementation. Previews are yet to be done.